pkg/workloadmeta/collectors/internal/docker/docker.go

```diff
@@ -415,3 +416,20 @@ func extractStatus(containerState *types.ContainerState) workloadmeta.ContainerS
 
 	return workloadmeta.ContainerStatusUnknown
 }
+
+func extractHealth(containerHealth *types.Health) workloadmeta.ContainerHealth {
```

Comment:
Could we add some unit tests for this?

Reply:
Unfortunately, this collector has no unit test and it's out of scope of this PR to add unit testing to this collector.